### PR TITLE
split routes up

### LINKED ISSUE #83: refactor router for server side rendering

### PR DESCRIPTION
**Pull request by jennschiffer**
splits up the router into smaller modules, allowing code splitting points within the app for setting up serverside rendering.

closes #83

### CODE REVIEW

File: src/routes/root.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import ServiceListContainer from '../components/service-list-container';
+import auth from '../service/auth';
+import serviceRoute from './service';
+
+export default {
+  path: '/',
+  component: Layout,
+  onEnter: auth.requireAuth,
+  getChildRoutes(location, cb) {
+    require.ensure([], require => {
```
Comment (tkellen) [inline src/routes/root.js]: can we get a comment here explaining what this is doing for newcomers?

**Comment by tkellen** [comment]
I really like that this is de-structuring the routes to POJOs instead of JSX.

**Comment by jennschiffer** [comment]
yeah this literally made the router make sense to me now

**Comment by tkellen** [comment]
:+1: to squash and merge

File: src/app.jsx [hunk 2]
```diff
@@ -4,28 +4,12 @@
...
-    <Route path="/login" component={Login} onEnter={auth.requireNoAuth} />
-    <Route path="/" component={Layout} onEnter={auth.requireAuth}>
-      <IndexRoute component={ServiceListContainer} />
-      <Route component={ServiceSubpage}>
-        <Route path="service/:slug" component={ServiceDetailsContainer} />
-      </Route>
-      <Route path="*" component={PageNotFound} />
-    </Route>
-  </Router>
+const renderRoutes = (
```
Comment (tkellen) [inline src/app.jsx]: can this be destructured to a pojo too?

**Pull request merged**